Update docs for version 0.9.9 (#278)
AequilibraE/qaequilibrae: docs/source/menus_in_detail/tripdistribution.rst

or *Save and use*. Case you choose to save, the vector will be saved in a temporary QGIS folder.

.. image:: ../images/tripdistribution-ipf-2.png
    :width: 513
    :align: center
    :alt: ipf_2

After the vector is properly loaded, it will appear in the *Load datasets* tab.

.. image:: ../images/tripdistribution-ipf-3.png
    :width: 513
    :align: center
    :alt: ipf_3

You can now select the production/attraction (origin/destination) vectors.

.. image:: ../images/tripdistribution-ipf-4.png
    :width: 513
    :align: center
    :alt: ipf_4

And the impedance matrix to be used.

.. image:: ../images/tripdistribution-ipf-5.png
    :width: 513
    :align: center
    :alt: ipf_5

To run the procedure, simply queue the job (and select the where the output file will be saved) 
and press *Run jobs*.

.. image:: ../images/tripdistribution-ipf-6.png
    :width: 513
    :align: center
    :alt: ipf_6

After the job is completed, a new window showing its procedure report will open.

.. image:: ../images/tripdistribution-ipf-7.png
    :width: 513
    :align: center
    :alt: ipf_7

.. note::

>     Although IPF aim is to balance the production/attraction vectors, the total sum of production 

Reviewer: ```suggestion
Production and Attraction vectors must be balanced prior to running IPF. 
```

>     and attraction must be the same, otherwise IPF cannot be run.

Reviewer: ```suggestion

```
